packages/libs/eda/src/lib/map/analysis/mapTypes/MapTypeHeaderStudyDetails.tsx

```diff
@@ -0,0 +1,178 @@
+    outputEntityCount.error && console.error(outputEntityCount.error);
+    studyEntityCount.error && console.error(studyEntityCount.error);
+    return (
+      <div>
+        <em>Could not load counts for map data</em>
+      </div>
+    );
+  }
+
+  if (
+    studyEntityCount.data == null ||
+    studyEntityCount.isFetching ||
+    outputEntityCount.data == null ||
+    outputEntityCount.isFetching
+  )
+    return (
+      <div
+        css={{
+          padding: '1em',
+          fontSize: '1.2em',
+          display: 'flex',
+          gap: '1ex',
+          alignItems: 'center',
+        }}
+      >
+        Loading counts&hellip;
+      </div>
+    );
+
+  const totalCountFormatted = format(outputEntityCount.data.count);
+  const totalStudiesFormatted = format(studyEntityCount.data.count);
+
+  const tooltipContent = (
+    <div>
+      <p>
+        {totalCountFormatted}{' '}
+        {outputEntityCount.data.count === 1
+          ? outputEntity.displayName + ' is'
+          : outputEntity.displayNamePlural + ' are'}{' '}
+        currently visualized on the map using markers. These are the{' '}
+        {outputEntity.displayNamePlural} that
+        <ul css={{ padding: '1em 0' }}>
+          <li>satisfy all your filters</li>
+          {includesTimeSliderFilter && (
+            <li>satisfy the time range you have selected</li>
+          )}
+          <li>
+            have data for the variable that is currently displayed on the
+            visible or slected markers
+          </li>
+          <li>
+            have appropriate values, if the marker has been custom-configured
+          </li>
+        </ul>
+      </p>
+      {onShowStudies && studyEntity && (
+        <p>
+          The visualized data comes from {totalStudiesFormatted}{' '}
+          {studyEntityCount.data.count === 1
+            ? studyEntity.displayName
+            : studyEntity.displayNamePlural}{' '}
+          <button type="button" onClick={() => onShowStudies(true)}>
+            Show list
+          </button>
+        </p>
+      )}
+    </div>
+  );
+
+  return (
+    <>
+      <div
+        css={{
+          padding: '1em',
+          fontSize: '1.2em',
+          display: 'flex',
+          gap: '1ex',
+          alignItems: 'center',
+        }}
+      >
+        {totalCountFormatted}{' '}
+        {outputEntityCount.data.count === 1
+          ? outputEntity.displayName
+          : outputEntity.displayNamePlural}
+        {onShowStudies && studyEntity && (
+          <>
+            {' '}
+            from {format(studyEntityCount.data.count)}{' '}
+            {studyEntity.displayNamePlural}
+          </>
+        )}
+        <Tooltip title={tooltipContent} interactive style={{ width: 'auto' }}>
+          <Info style={{ color: '#069', height: '.8em', width: '.8em' }} />
+        </Tooltip>
+      </div>
+    </>
+  );
+}
+
+function useEntityCount(entityId?: string, filters?: Filter[]) {
```

Comment:
did you mean to put this somewhere central?

Reply:
Eventually, I would like to replace the other hook with this.